customizations/README.md

```diff
@@ -1,122 +1,107 @@
   ```yaml
   appConfigValues:
     selfHostedDomain: "my.domain.com"
+  ```
 
-  appSecrets:
-    googleMapsApiKey:
-      value: "<google-maps-api-key>"
+And add the following at the end of ALL the `helm install` or `helm upgrade` command:
 
-  router:
-    service:
-      type: LoadBalancer
-  ```
-  And add the following at the end of ALL the install or upgrade command:
-  ```bash
-  ... -f customization.yaml
-  ```
-- Use the parameters as arguments. You can specify each parameter using the `--set key=value[,key=value]` argument. For example, add the following at the end of ALL the install or upgrade command:
   ```bash
-  ... --set appConfigValues.selfHostedDomain=my.domain.com \
-    --set appSecrets.googleMapsApiKey.value=<google-maps-api-key> \
-    --set router.service.type=LoadBalancer
+  helm instal .. -f client-conf.yaml
   ```
 
+You can also override values through the command-line to `helm`. Adding the argument: `--set key=value[,key=value]`
+
+## Available Configurations
 
-## Configure the domain of your self-hosted
+There are several things that you can configure in you CARTO Self Hosted:
 
-The most important step to have your CARTO self-hosted ready to be used is to configure the domain to be used.
+### Configure the domain of your Self Hosted
 
-> ⚠️ CARTO self-hosted is not designed to be used in the path of a URL, it needs a full domain or subdomain. ⚠️
+The most important step to have your CARTO Self Hosted ready to be used is to configure the domain to be used.
+
+> ⚠️ CARTO Self Hosted is not designed to be used in the path of a URL, it needs a full domain or subdomain. ⚠️
+
+To do this you need to [add the following customization](#how-to-apply-the-configurations):
 
-To do this you need to [add the following customization](#how-to-define-customizations):
 ```yaml
 appConfigValues:
   selfHostedDomain: "my.domain.com"
+appSecrets:
+  #googleMapsApiKey:
+  #value: "<google-maps-api-key>"
+  #other secrets, like buckets' configuration
 ```
 
 Don't forget to upgrade your chart after the change.
 
+### Access to CARTO from outside the cluster
 
-## Access to CARTO from outside the cluster
-By default, the `router` Service of CARTO self-hosted is configured in mode `ClusterIP` so it's only usable from inside the cluster.
-To access to it you can exec locally a [kubectl port-forward](https://kubernetes.io/docs/tasks/access-application-cluster/port-forward-access-application-cluster/) but this only make it accessible to your machine.
+The entry point to the CARTO Self Hosted is through the `router` Service. By default it is configured in `ClusterIP` mode. That means it's
+only usable from inside the cluster. If you want to connect to your deployment with this mode, you need to use [kubectl port-forward]
+(<https://kubernetes.io/docs/tasks/access-application-cluster/port-forward-access-application-cluster/>). But this only makes it accessible
```

Comment:
ey, that link is broken with that format

Reply:
the problem was the new line